examples/image.md

```diff
@@ -0,0 +1,237 @@
+
+```cpp
+#include <mlboard/mlboard.hpp>
+#include <iostream>
+#include <chrono> 
+#include <ctime> 
+#include <future>
+
+int main()
+{
+    std::chrono::time_point<std::chrono::system_clock> start, end; 
+    start = std::chrono::system_clock::now(); 
+    mlboard::FileWriter f1("temp");
+    
+    ifstream fin("./assets/multiple_image.jpg", ios::binary);
+    ostringstream ss;
+    vector<string>encodedImages;
+    ss << fin.rdbuf();
+    encodedImages.push_back(ss.str());
+    fin.close();
+    fin.open("./assets/single_image.jpg", ios::binary);
+    ss << fin.rdbuf();
+    encodedImages.push_back(ss.str());
+    ss.str("");
+    fin.close();
+    mlboard::SummaryWriter<mlboard::FileWriter>::Image(
+         "Test Image", 1, encodedImages, 512, 512, f1, "Sample Image",
+         "This is a Sample image logged using mlboard.");
+    f1.Close();
+    end = std::chrono::system_clock::now(); 
+    std::chrono::duration<double> elapsed_seconds = end - start; 
+    std::time_t end_time = std::chrono::system_clock::to_time_t(end); 
+    
+    std::cout << "finished computation at " << std::ctime(&end_time) 
+              << "elapsed time: " << elapsed_seconds.count() << "s\n"; 
+}
+```
+
+The output would be similar to: 
+
+<p>
+<img src = "assets/multiple_image.jpg" width = "800" height = "400"/>
+</p>
+
+### 3. Multiple Images Arma Mat
+
+Since we would be logging mlpack's metrics there are chances when we would have to visualize images stored in `arma::mat` and thus you can use the following API to directly log Armadillo matrices:
+
+```cpp
+void SummaryWriter<Filewriter>::Image(const std::string& tag,
+                                      int step,
+                                      arma::Mat<eT>& matrix,
+                                      mlpack::data::ImageInfo& info,
+                                      Filewriter& fw,
+                                      const std::string& displayName,
+                                      const std::string& description)
+```
+
+The API accepts `tag`, `step`, `encodedImage` (An `arma::mat` which has image data) and then the `mlpack::data::ImageInfo` object along with filewriter object and Name and description.
+
+Following is a snippet that would log a multiple image which is stored in `arma::mat` for 1 step in temp directory.
+
+```cpp
+#include <mlboard/mlboard.hpp>
+#include <iostream>
+#include <chrono> 
+#include <ctime> 
+#include <future>
+
+int main()
+{
+    std::chrono::time_point<std::chrono::system_clock> start, end; 
+    start = std::chrono::system_clock::now(); 
+    mlboard::FileWriter f1("temp");
+    
+    // Following line are just there to come up with a arma::mat of images.
+    // If you aldeardy have a matrix with images you can avoid the following lines.
+    arma::Mat<unsigned char> matrix;
+    mlpack::data::ImageInfo info;
+    std::vector<std::string> files = {"./assets/single_image.jpg",
+         "./assets/mltiple_image.jpg"};
```

Comment:
```suggestion
         "./assets/single_image.jpg"};
```

Reply:
mlpack::data::save("image") does not support saving of two different dimension images

Reply:
Oh I see.

Reply:
Hi @jeffin143, can we avoid using mlpack::data::save() and load() in any way? Have you tried arma::raw_print function you discussed some time before?

Reply:
Even if we have to use arma::raw_print() , we have to get the matrix with image data , and that we can only get using Mlpack::data::load() , I am not sure if we can avoid that 

I tried with arma::raw_print , it didn't work out and hence I need tk resort to internall saving the images of matrix 


But to get the matrix with image data we have to use arma::Mlpack::load

Reply:
Okay I will take a closer look tommorow.